src/related-items-and-comparisons/RelatedItemsList.jsx

```diff
@@ -31,12 +31,14 @@ const RelatedItemsList = (props) => {
       <div className="related-container">
         {leftButton}
         <div className="related-items-list">
-          {currList.map((product) => <CardItem key={product.id} product={product} />)}
+          {currList.map(({ product, metadata, styles }) =>
```

Comment:
@jonathanjewett Your info for each item now includes metadata and styles.

Reply:
Thanks, yeah it's probably better to have too much than too little, I can always filter what I need.